aesara/graph/destroyhandler.py

```diff
@@ -305,7 +305,7 @@ def __init__(self, do_imports_on_attach=True, algo=None):
         TODO: change name to var_to_vroot.
 
         """
-        self.droot = OrderedDict()
+        self.droot = dict()
```

Comment:
fyi the comparison between `dict` is still not order-dependent while the comparison between `OrderedDict` is. I don't know if we'll need that property, but it's good to know anyway.

```python
OrderedDict([(1,1), (2,2)]) == OrderedDict([(2,2), (1,1)]) 
# False
dict([(1,1), (2,2)]) == dict([(2,2), (1,1)]) 
# True
```

Reply:
Yes, I'm aware.